quantized_mesh_encoder/ecef.py

```diff
@@ -16,22 +16,22 @@ def to_ecef(positions):
 
     Args:
         - positions: expected to be an ndarray with shape (-1, 3)
-    from latitude-longitude-height to
+        - ellisoid: a dict that defines an ellipsoid with with "a" and "b" values. Defaults to WGS84
```

Comment:
Just use the same docstring as you have in `encode`, and you have a typo here: `ellisoid -> ellipsoid`.